src/openvic-simulation/military/UnitInstance.cpp

```diff
@@ -0,0 +1,29 @@
+#include "UnitInstance.hpp"
+#include "military/UnitType.hpp"
+#include <vector>
+
+using namespace OpenVic;
+
+RegimentInstance::RegimentInstance(std::string_view new_name, RegimentType const& new_regiment_type, Pop const& new_pop)
+	: UnitInstance { new_name, new_regiment_type }, pop { new_pop } {}
+
+ShipInstance::ShipInstance(std::string_view new_name, ShipType const& new_ship_type)
+	: UnitInstance { new_name, new_ship_type } {}
+
+//TODO: pathfinding logic
+MovementInfo::MovementInfo(Province const* starting_province, Province const* target_province)
+	: path { std::vector { starting_province, target_province }}, movement_progress { 0 }, locked { false } {}
```

Comment:
```suggestion
	: path { std::vector { starting_province, target_province } }, movement_progress { 0 } {}
```
`locked` removed + space between the `}`s